RNDFPBanner.js

```diff
@@ -14,18 +14,14 @@ export default class DFPBanner extends React.Component {
     this.onSizeChange = this.onSizeChange.bind(this);
     this.state = {
       style: {
-        width: 1,
-        height: 1,
+        flex: 1,
       },
     };
   }
 
   onSizeChange(event) {
```

Comment:
Don't you need a arrow function here to remove the `.bind` in the render ? Or is that one of the basic lifecycle methods ?

Reply:
I asked that too - it's already `.bind`-ed in the constructor

Reply:
Oh indeed. I missed that. I don't like the bind in the constructor, but whatever.